### PR TITLE
Add docker compose.yml file generation

### PR DESCRIPTION
**Pull request by hamzajg**

### CODE REVIEW

**Comment by VaughnVernon** [comment]
@hamzajg Cool. Does this include images and configurations for all applicable options:

- RabbitMQ 
- Command Model Database
- Query Model Database 
- Others?

**Comment by hamzajg** [comment]
> @hamzajg Cool. Does this include images and configurations for all applicable options:
> 
> * RabbitMQ
> * Command Model Database
> * Query Model Database
> * Others?

For now only RabbitMQ

File: src/test/resources/text-expectations/java/docker-compose-with-postgres-mysql.text [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      POSTGRES_HOST_AUTH_METHOD: "trust"
+
+  mysql:
+    image: mysql:5.7
+    container_name: mysql
+    volumes:
+      - 'mysql'
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+      MYSQL_DATABASE: xoomapp-query
```
Comment (VaughnVernon) [inline src/test/resources/text-expectations/java/docker-compose-with-postgres-mysql.text:38]: @hamzajg Please name: xoomapp-query-model

**Review by VaughnVernon — CHANGES_REQUESTED** [review]
@hamzajg Cool! 👍🏻 Please see comments for database naming changes.

**Pull request merged**